server/janitor/janitor.go

```diff
@@ -1,79 +1,180 @@
 package janitor
 
 import (
+	"context"
 	"flag"
+	"strings"
 	"time"
 
 	"github.com/buildbuddy-io/buildbuddy/server/environment"
 	"github.com/buildbuddy-io/buildbuddy/server/tables"
+	"github.com/buildbuddy-io/buildbuddy/server/util/db"
 	"github.com/buildbuddy-io/buildbuddy/server/util/log"
 )
 
 var (
+	// Flags shared by both invocation and execution janitor.
+	logDeletionErrors = flag.Bool("log_deletion_errors", false, "If true; log errors when ttl-deleting expired data")
+
+	// Flags for Invocaction Janitor.
 	ttlSeconds = flag.Int("storage.ttl_seconds", 0, "The time, in seconds, to keep invocations before deletion. 0 disables invocation deletion.")
 
-	cleanupBatchSize  = flag.Int("storage.cleanup_batch_size", 10, "How many invocations to delete in each janitor cleanup task")
-	cleanupInterval   = flag.Duration("cleanup_interval", 10*60*time.Second, "How often the janitor cleanup tasks will run")
-	cleanupWorkers    = flag.Int("cleanup_workers", 1, "How many cleanup tasks to run")
-	logDeletionErrors = flag.Bool("log_deletion_errors", false, "If true; log errors when ttl-deleting expired data")
+	cleanupBatchSize = flag.Int("storage.cleanup_batch_size", 10, "How many invocations to delete in each janitor cleanup task")
```

Comment:
Can you add an 'invocation' prefix to the flag variables? We can leave the flag names as-is.

Reply:
Done